cpp/ql/src/Best Practices/Likely Errors/CommaBeforeMisleadingIndentation.ql

```diff
@@ -26,6 +27,12 @@ predicate isInLoopHead(CommaExpr ce) {
   ce.getEnclosingStmt() = any(ForStmt f).getInitialization()
 }
 
+predicate isInDecltypeOrSizeof(CommaExpr ce) {
```

Comment:
Could this predicate just be defined as `ce.isUnevaluated()`? That certainly includes the two cases you've given here, along with operands of `noexcept`, `alignof` and `typeid` (which are all surrounded by parentheses).